docs/Testing/OSXWinObjCTests.md

```diff
@@ -47,9 +47,9 @@ While it's possible to checkout a separate git enlistment on an OSX machine and
 #### Current known failures:
 
 
-    * [==========] 821 tests from 74 test cases ran. (56608 ms total)
-    * [  PASSED  ] 698 tests.
-    * [  FAILED  ] 123 tests, listed below:
+    * [==========] 833 tests from 74 test cases ran. (58373 ms total)
```

Comment:
Agreed on both fronts.

---
In reply to: [73249644](https://github.com/Microsoft/WinObjC/pull/792#discussion_r73249644) [](ancestors = 73249644,73249616)